src/options.rs

```diff
@@ -74,6 +80,8 @@ impl Default for ColumnOptions {
 			preimage: false,
 			uniform: false,
 			ref_counted: false,
+			compression: crate::compress::CompressType::NoCompression,
+			compression_treshold: usize::max_value(),
```

Comment:
using platform specific`usize` here may create issues when moving databases between machines. I'd suggest using `u32` for the type

Reply:
Also let's use some sensible value for the default, like 4kbytes.

Reply:
Should have been 0 in the first place (the original intent was not applied). Just got a suprise when re-checking column 8 :(
4k will be good.